src/Tizen.NUI/src/public/WebView/WebViewPageLoadEventArgs.cs

```diff
@@ -16,22 +16,21 @@
  */
 
 using System;
-using System.ComponentModel;
 
 namespace Tizen.NUI
 {
     /// <summary>
     /// Event arguments that passed via the WebView.PageLoadStarted or WebView.PageLoadFinished.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 8 </since_tizen>
     public class WebViewPageLoadEventArgs : EventArgs
     {
-
         private WebView _webView;
+
         /// <summary>
         /// The view for displaying webpages.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public WebView WebView
```

Comment:
I think this will not be necessary : WebViewPageLoadEventArgs.WebView

In C#, all event callbacks have a first argument `sender` which is `WebView` in WebView event

Reply:
removed.